cookbook/app/views/ingredients/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_for @ingredient do |f| %>
+  <p>
+      <%= f.label :name %><br>
+      <%= f.text_field :name %>
+    </p>
+
+    <p>
+      <%= f.label :img_url %><br>
+      <%= f.text_area :img_url %>
+    </p>
+
+    <p>
+      <%= f.label :price %><br>
+      <%= f.text_area :price %>
+    </p>
+<p>Select a recipe <%= f.select :recipe_id, [['Birthday Cake', 1],['Pasta Dish', 2]] %></p>
+    <p>
```

Comment:
In place of passing in a hard-coded array of 2-item arrays, you can pass in an instance variable, perhaps named, `@recipes_for_select` defined in your `controllers/recipes_controller.rb`as `@recipes_for_select = Recipe.all.map { |recipe| [recipe.name, recipe.id] }